resolwe_bio/tests/processes/test_support_processors.py

```diff
@@ -289,3 +289,71 @@ def test_seqtk_sample(self):
                          compression='gzip')
         self.assertFiles(seqtk_paired, 'fastq2', ['seqtk_subsampled_reads_paired_end_mate2.fastq.gz'],
                          compression='gzip')
+
+    @tag_process('spikein-qc')
+    def test_spikein_pairwise(self):
+        with self.preparation_stage():
+            expression_1 = self.prepare_expression(
+                f_exp='exp1_cpm_ercc_sirv.tab.gz',
+                f_type='CPM',
+                name='Sample',
+                source='ENSEMBL',
+                species='Homo sapiens')
+            expression_2 = self.prepare_expression(
+                f_exp='exp2_cpm_ercc_sirv.tab.gz',
+                f_type='CPM',
+                name='Replicate 1',
+                source='ENSEMBL',
+                species='Homo sapiens')
+            expression_3 = self.prepare_expression(
+                f_exp='exp3_cpm_ercc_sirv.tab.gz',
+                f_type='CPM',
+                name='Replicate 2',
+                source='ENSEMBL',
+                species='Homo sapiens')
+            expression_4 = self.prepare_expression(
+                f_exp='exp4_cpm_ercc_sirv.tab.gz',
+                f_type='CPM',
+                name='Sample without ERCC',
+                source='ENSEMBL',
+                species='Homo sapiens')
+
+        inputs = {
+            'sample': expression_1.pk,
+            'mix': 'sirv_set3',
+        }
+
+        wo_replicates = self.run_process('spikein-qc', inputs)
+        # self.assertFiles(sirv, 'plots', [
```

Comment:
This is not the best practice. As Barbara already suggested, we should support `assedtFilesExist` method in resolwe, similar to the existing [`assertFileExists`](https://github.com/genialis/resolwe/blob/9307c15801d4aff0a32d453cb4366bdb2c2200e8/resolwe/test/testcases/process.py#L694), but for `list:basic:file` fields.

If we accept such workaround, it will stay in forever and at some point someone will copy it and unintentionally add even more code debt.

Reply:
I agree, but this will have to come in with the next release.